### PR TITLE
NMS-14182: more alarmd doc tweaks

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14182

### CODE REVIEW

**Comment by mmahacek** [comment]
Extension of items missed in https://github.com/OpenNMS/opennms/pull/4557

File: docs/modules/operation/pages/alarms/configuring-alarms.adoc [hunk 1]
```diff
@@ -69,15 +69,17 @@ Most commonly, the event's identifier (UEI) is used as the left-most (least sign
     <alarm-data reduction-key="%uei%:%dpname%:%nodeid%" alarm-type="1" auto-clean="false"/>
 </event>
 ----
+The `$dpname%` portion of the key refers to the "distributed poller name", which would be the name of the Minion that originated the event.
 ====
 
 .Least Significant reduction-key Attribute
 ====
-TIP: Decreasing the significance of the reduction-key is a way to aggregate, for example, all nodes down in to a single alarm.
+Decreasing the significance of the reduction-key is a way to aggregate, for example, all nodes down in to a single alarm.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/alarms/configuring-alarms.adoc:77]:
```suggestion
Decreasing the significance of the reduction-key is a way to aggregate, for example, all nodes into a single alarm.
```

File: docs/modules/operation/pages/alarms/configuring-alarms.adoc [hunk 2]
```diff
@@ -89,12 +91,13 @@ However, the UEI `uei.opennms.org/nodes/nodeUp` would *not* be a good 'pairwise'
...
+Alarmd is designed to automatically match resolving events with an existing alarm.
+Alarms with matching resolutions with problems (Ups with Downs), should be indicated with the alarm-type attribute.
 There are currently three types of alarms:
 
-* problem (1)
-* resolution (2)
-* notification (3)
+* alarm-type="1" (problem alarm)
+* alarm-type="2" (resolving alarm)
+* alarm-type="3" (notification alarm... alarm with no resolution such as SNMP Authentication Failures)
```
Comment (Bonrob2) [inline docs/modules/operation/pages/alarms/configuring-alarms.adoc:100]:
```suggestion
* alarm-type="3" (notification alarm ... alarm with no resolution, such as SNMP authentication failures)
```

File: docs/modules/operation/pages/alarms/configuring-alarms.adoc [hunk 3]
```diff
@@ -104,6 +107,29 @@ NOTE: If an alarm transitions from an alarm-type 2 back to alarm-type 1, the sev
 This attribute is used in the pairwise correlation feature of Alarmd.
 When configuring a resolution alarm, set this attribute to match the reduction-key of the corresponding problem alarm.
 
+.Example of a `interfaceUp` event clearing a `interfaceDown` alarm
```
Comment (Bonrob2) [inline docs/modules/operation/pages/alarms/configuring-alarms.adoc:110]:
```suggestion
.Example of an `interfaceUp` event clearing an `interfaceDown` alarm
```

File: docs/modules/operation/pages/alarms/configuring-alarms.adoc [hunk 4]
```diff
@@ -104,6 +107,29 @@ NOTE: If an alarm transitions from an alarm-type 2 back to alarm-type 1, the sev
...
+    <event>
+      <uei>uei.opennms.org/nodes/interfaceUp</uei>
+      ...
+      <alarm-data reduction-key="%uei%:%dpname%:%nodeid%:%interface%"
+                  alarm-type="2"
+                  clear-key="uei.opennms.org/nodes/interfaceDown:%dpname%:%nodeid%:%interface%"<2>
+                  auto-clean="false"/>
+   </event>
+----
+<1> The `interfaceDown` event sets a `reduction-key` that include enough info to identify a specific interface on a specific node.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/alarms/configuring-alarms.adoc:130]:
```suggestion
<1> The `interfaceDown` event sets a `reduction-key` that includes enough information to identify a specific interface on a specific node.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
minor changes. Thanks!

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Pull request merged**